index.html

```diff
@@ -1,12 +1,144 @@
+                </form>
+            </div>
+            <div id="end">
+                <span><i class="ico fa fa-video"></i></span>
+                <span><i class="ico fa fa-bell"></i></span>
+                <span><i class="ico fa fa-user"></i></span>
+            </div>
+        </nav>
+    </section>
+    <section>
+        <main>
+            <div id="primary">
+                <iframe width="100%" height="480" src="https://www.youtube.com/embed/MmPIyZ7u6Gk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                <h1 class="yt-title">Level Up Experience | Operators</h1>
+                <div class="yt-video-icons">
+                    <p>1,195 views . 1 Apr 2020</p>
+                    <div>
+                        <span><i class="fa yt-vd-icons fa-thumbs-up"></i>100</span>
+                        <span><i class="fa yt-vd-icons fa-thumbs-down"></i>Dislike</span>
+                        <span><i class="fa yt-vd-icons fa-share"></i>Share</span>
+                        <span><i class="fa yt-vd-icons fa-scissors"></i>Clip</span>
+                        <span><i class="fa yt-vd-icons plus-minus"></i>Save</span>
+                        <span class="yt-vd-icons">...</span>
+                    </div>
+                </div>
+            </div>
+            <div id="secondary">
+                <div class="ytv-popular-videos">
+                    <button class="popular-video-btn active">All</button>
+                    <button class="popular-video-btn">E-cormerce</button>
+                    <button class="popular-video-btn">Sales</button>
+                    <button class="popular-video-btn">Related</button>
+                </div>
+                <div class="secondary">
+                    <div class="yt-suggested-videos">
+                        <img src="img/mqdefault_6s (1).webp" alt="">
+                        <div class="yt-suggested-videos-details">
+                            <h3 class="ytv-title">In Love with my school Teacher</h3>
+                            <p class="ytv-account">JAYGWEN TV</p>
+                            <p class="ytv-desc">18k Views . 22days ago</p>
+                        </div>
+                    </div>
+                    <div class="yt-suggested-videos">
+                        <img src="img/mqdefault_6s (2).webp" alt="">
+                        <div class="yt-suggested-videos-details">
+                            <h3 class="ytv-title">The Only Shoppify Dropshipping you will ever need</h3>
+                            <p class="ytv-account">Nathan Nazareth</p>
+                            <p class="ytv-desc">122k Views . 1 Month ago</p>
+                        </div>
+                    </div>
+                    <div class="yt-suggested-videos">
+                        <img src="img/mqdefault_6s (3).webp" alt="">
```

Comment:
nice! if you can make your image names real and also not leave spaces in between the names. 
Also, add what the image is about in the alt attribute, it is a good practice to make your website more accessible on screen readers, help visually impaired etc.

Reply:
Nice one :handshake: